go-controller/cmd/ovn-k8s-cni-overlay/app/helper_windows.go

```diff
@@ -0,0 +1,221 @@
+// +build windows
+
+package app
+
+import (
+	"encoding/json"
+	"fmt"
+	"net"
+	"os/exec"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/Microsoft/hcsshim"
+	"github.com/containernetworking/cni/pkg/skel"
+	"github.com/containernetworking/cni/pkg/types/current"
+
+	"github.com/openvswitch/ovn-kubernetes/go-controller/pkg/config"
+)
+
+// noNameNetNS - This is received for the infra container, in this case we
+// have to create the network endpoint and attach it to the infra container
+// containerNetNS - This is received for all the other containers from
+// the same pod, do not create the network endpoint in thi case
+const (
+	noNameNetNS    = "none"
+	containerNetNS = "container:"
+)
+
+// More details about the above constants can be found in the following PR:
+// https://github.com/kubernetes/kubernetes/pull/51063
+
+/**
+ * When the HNS Endpoint is created, it asks for the HNS Id in order to
+ * attach it to the desired network. This function finds the HNS Id of the
```

Comment:
id->ID

Reply:
I wanted to keep the same name as the one from HNSNetwork structure:
```
// HNSNetwork represents a network in HNS
type HNSNetwork struct {
	Id                   string            `json:"ID,omitempty"`
	Name                 string            `json:",omitempty"`
	Type                 string            `json:",omitempty"`
	...........................................
}
```
I think HNS Id it's good enough.

Reply:
Fair enough.